dht_test.go

```diff
@@ -708,6 +711,7 @@ func TestRefreshBelowMinRTThreshold(t *testing.T) {
 
 	host, err := bhost.NewHost(swarmt.GenSwarm(t, swarmt.OptDisableReuseport), new(bhost.HostOpts))
 	require.NoError(t, err)
+	host.Start()
```

Comment:
No cleanup here ? This also applies to all new `.Start` calls.

Do theses hosts need to be stopped ?

Reply:
`.Close` does it. And these tests should have already been calling `.Close`

Reply:
Looks like it's missing, anyway tests are passing and the hosts are killed when the test process exit anyway that fine enough.